www/cobrands/groundcontrol/templates/en/photo.html

```diff
@@ -10,6 +10,27 @@ <h1>Add Photos</h1>
         <% } %>
 </div>
 
+<div class="pothole-size-selector hidden">
+    <h2>Size of pothole?</h2>
+    <div class="photo"></div>
+    <div class="photo-pothole-size">
+        <% Object.entries(CONFIG.POTHOLE_SIZES).forEach(function(size, index) { %>
```

Comment:
If it works, no issue I guess, but just to note I don't think `entries` order is actually defined (though presumably all are doing order given by default).